shell/src/main/java/alluxio/cli/fs/command/DistributedCpCommand.java

```diff
@@ -145,8 +145,7 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
     AlluxioConfiguration conf = mFsContext.getPathConf(dstPath);
     mWriteType = conf.get(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT);
-    int defaultBatchSize = conf.getInt(PropertyKey.JOB_REQUEST_BATCH_SIZE);
-    int batchSize = FileSystemShellUtils.getIntArg(cl, BATCH_SIZE_OPTION, defaultBatchSize);
```

Comment:
Just for context, can you explain what the fix does? `PropertyKey.JOB_REQUEST_BATCH_SIZE` had a default value of 20 so what does setting the default to `1` do? Thanks!

Reply:
If it's 1 then it would send the Migrate request(original implementation) instead of the batched job request